### DISCUSSION TITLE
FR: define size of quick terminal

### DISCUSSION THREAD

**Discussion by jjt** [ghostty-org/ghostty#3339] [Feature Requests, Ideas] [answered]
this is too squat and wide, it does not spark joy

![image](https://github.com/user-attachments/assets/2ce5986a-f5f7-4bc2-a965-679d5e218b59)

this is handsome and perfect, it sparks joy

![image](https://github.com/user-attachments/assets/ab502721-84a3-4929-9773-ae70d272cce4)

**Comment by rhodes-b** [comment, marked as answer]
duplicate #2384

**Comment by jjt** [comment]
i searched discussions but not issues 🙃

closing as duplicate
